pkg/policyreport/reportrequest.go

```diff
@@ -86,6 +82,7 @@ func NewReportChangeRequestGenerator(client *policyreportclient.Clientset,
 		polListerSynced:                  polInformer.Informer().HasSynced,
 		queue:                            workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), workQueueName),
 		dataStore:                        newDataStore(),
+		requestCreator:                   newChangeRequestCreator(dclient, 3*time.Second, log.WithName("requestCreator")),
```

Comment:
I set the default ticker interval to 3 seconds (the `changeReqeustCreator` will aggregate buffered requests every 3s). Should we allow this to be tuned dynamically across various loads of clusters?

Reply:
So, if there are no buffered requests will there be an overhead? 

Why not use a fixed size pool of workers that block on the queue instead?

Reply:
> if there are no buffered requests will there be an overhead
No.

We can use a fixed size pool, but also need to set a "timeout" interval to flush the queue. So I buffer the requests and aggrate them every fixed interval.